File: app/src/main/java/com/kickstarter/mock/factories/ItemFactory.java
package com.kickstarter.mock.factories;

import com.kickstarter.models.Item;

import androidx.annotation.NonNull;

public final class ItemFactory {
  private ItemFactory() {}

  public static @NonNull Item item() {
    return Item.builder()
      .amount(10.0f)
      .id(IdFactory.id())
      .name("T-Shirt")
      .projectId(IdFactory.id())
      .build();
  }
}

File: app/src/main/java/com/kickstarter/mock/factories/ItemFactory.kt
package com.kickstarter.mock.factories

import com.kickstarter.models.Item
import com.kickstarter.models.Item.Companion.builder

object ItemFactory {
    fun item(): Item {
        return builder()
            .amount(10.0f)
            .id(IdFactory.id().toLong())
            .name("T-Shirt")
            .projectId(IdFactory.id().toLong())
            .build()
    }
}
